### PR TITLE
Rewrite of the Record  Layer Description

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

**Comment by hannestschofenig** [comment]
Trying to address https://github.com/tlswg/dtls-conn-id/issues/17 but also trying to expand the description  a bit for clarity.

File: draft-ietf-tls-dtls-connection-id.md [hunk 1]
```diff
@@ -202,31 +185,79 @@ two implications:
...
 ~~~~
+	 struct {
+         ContentType type;
+         ProtocolVersion version;
+         uint16 epoch;                         // DTLS field
+         uint48 sequence_number;               // DTLS field
+         uint16 length;
+         opaque fragment[DTLSPlaintext.length];
+     } DTLSPlaintext;
+
```
Comment (ekr) [inline draft-ietf-tls-dtls-connection-id.md]: Please fix these tabs. We use spaces.

File: draft-ietf-tls-dtls-connection-id.md [hunk 2]
```diff
@@ -202,31 +185,79 @@ two implications:
...
          ContentType type;
          uint8 zeros[length_of_padding];
-      } DTLSWrappedCompressed;
-~~~~
+     } DTLSInnerPlaintext;
 
+     struct {
+         ContentType special_type = tls12_cid; /* 25 */
+         ProtocolVersion version;
+		 uint16 epoch;                         // DTLS field
```
Comment (ekr) [inline draft-ietf-tls-dtls-connection-id.md]: Rogue tab.

File: draft-ietf-tls-dtls-connection-id.md [hunk 3]
```diff
@@ -202,31 +185,79 @@ two implications:
...
+   the following DTLSCiphertext.encrypted_record, which is the sum of 
+   the lengths of the content and the padding, plus one for the inner 
+   content type, plus any expansion added by the AEAD algorithm.    
+   
+encrypted_record
+:  The AEAD-encrypted form of the serialized DTLSInnerPlaintext structure.
+
+Other fields are defined in RFC 6347. Note that this specification does 
+not make use of the DTLSCompressed structure since the use of compression 
+has been discouraged.
```
Comment (ekr) [inline draft-ietf-tls-dtls-connection-id.md]: Also, you can't use it with DTLS anyway at this point

**Review by ekr — APPROVED** [review]

**Pull request merged**